nautobot/extras/views.py

```diff
@@ -481,8 +480,7 @@ def construct_custom_field_delete_tasks(self, queryset):
         Helper method to construct a list of celery tasks to execute when bulk deleting custom fields.
         """
         tasks = [
-            # 2.0 TODO: #824 use obj.slug instead of obj.name
-            delete_custom_field_data.si(obj.name, set(obj.content_types.values_list("pk", flat=True)))
+            delete_custom_field_data.si(obj.key, set(obj.content_types.values_list("pk", flat=True)))
```

Comment:
Since this shorthand isn't used anywhere else in the code

```suggestion
            delete_custom_field_data.signature(obj.key, set(obj.content_types.values_list("pk", flat=True)))
```